scalding-core/src/main/scala/com/twitter/scalding/CumulativeSum.scala

```diff
@@ -60,7 +65,10 @@ object CumulativeSum {
       implicit ordS: Ordering[S],
       sg: Semigroup[V],
       ordU: Ordering[U],
-      ordK: Ordering[K]): TypedPipe[(K, (U, V))] = {
+      ordK: Ordering[K],
+      tagK: ClassTag[K],
+      tagV: ClassTag[V],
+      tagU: ClassTag[U]): TypedPipe[(K, (U, V))] = {
 
       val sumPerS = pipe
         .map { case (k, (u: U, v: V)) => (k, partition(u)) -> v }
```

Comment:
actually, I think it is as easy as removing the `U` and `V` and not adding the ClassTags.